pymc4/distributions/discrete.py

```diff
@@ -128,6 +130,74 @@ def upper_limit(self):
         return self.conditions["total_count"]
 
 
+class BetaBinomial(BoundedDiscreteCompoundDistribution):
+    r"""Bounded Discrete compound Beta-Binomial Random Variable
+
+    The pmf of this distribution is
+
+    .. math:: f(x \mid n, \alpha, \beta) = n \choose x \frac{B(x + \alpha, n - x + \beta)}{B(\alpha, \beta)}
+
+    .. plot::
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+        import scipy.stats as st
+        plt.style.use('seaborn-darkgrid')
+        lows = [1, -2]
```

Comment:
Oh, shoot! Adopted docstring from uniform distribution but forgot to edit it! Let me do that real quick